lib/matplotlib/text.py

```diff
@@ -1838,9 +1838,12 @@ def transform(renderer) -> Transform
                 self._arrow_relpos = arrowprops.pop("relpos", (0.5, 0.5))
             else:
                 # modified YAArrow API to be used with FancyArrowPatch
-                for key in [
-                        'width', 'headwidth', 'headlength', 'shrink', 'frac']:
+                for key in ['width', 'headwidth', 'headlength', 'shrink']:
                     arrowprops.pop(key, None)
+                if 'frac' in arrowprops:
+                    _api.warn_deprecated(
+                        "3.8", message="Support for passing the (unused) 'frac' key "
+                        "has been removed %(since)s and will be removed %(removal)s.")
```

Comment:
It's still only deprecated?
```suggestion
                if 'frac' in arrowprops:
                    _api.warn_deprecated(
                        "3.8", message="Support for passing the (unused) 'frac' key "
                        "has been deprecated %(since)s and will be removed %(removal)s.")
                    arrowprops.pop('frac')
```

Reply:
yup